src/api/apis/MobyGamesAPI.ts

```diff
@@ -21,6 +21,10 @@ export class MobyGamesAPI extends APIModel {
 	async searchByTitle(title: string): Promise<MediaTypeModel[]> {
 		console.log(`MDB | api "${this.apiName}" queried by Title`);
 
+		if(!this.plugin.settings.MobyGamesKey) {
+			throw Error(`MDB | MobyGames ${this.apiName} API key missing.`);
```

Comment:
Doesn't the `MobyGames ${this.apiName} API` cause duplicate names in the error message?